src/main/java/ru/l1ttleO/boyfriend/Actions.java

```diff
@@ -106,16 +101,11 @@ public static void muteMember(final @NotNull MessageChannel channel, final @NotN
         if (existingMute != null)
             existingMute.interrupt();
         if (duration > 0) {
-            final Thread thread = new Thread(() -> {
-                try {
-                    Thread.sleep(duration * 1000L);
-                    unmuteMember(null, role, guild.getSelfMember(), muted, "Время наказания истекло", silent);
-                } catch (final InterruptedException ignored) {
-                }
-            }, "Mute timer " + muted.getId());
-            guildMutes.put(muted.getIdLong(), thread);
+            final DelayedRunnable runnable = new DelayedRunnable(MUTES_THREAD_GROUP, (DelayedRunnable thisDR) -> {
+                unmuteMember(null, role, guild.getSelfMember(), muted, "Время наказания истекло", silent);
+            }, "Mute timer " + muted.getId(), duration * 1000L, null);
```

Comment:
```suggestion
            final DelayedRunnable runnable = new DelayedRunnable(MUTES_THREAD_GROUP, (DelayedRunnable thisDR) -> unmuteMember(null, role, guild.getSelfMember(), muted, "Время наказания истекло", silent),
                                                                 "Mute timer " + muted.getId(), duration * 1000L, null);
```
Reason: `Statement lambda can be replaced with expression lambda`